variables.tf

```diff
@@ -221,3 +221,8 @@ variable "permissions_boundary" {
   description = "If provided, all IAM roles will be created with this permissions boundary attached."
   default     = ""
 }
+
+variable "iam_path" {
+  description = "If provided, all IAM roles will be created on this path."
+  default     = ""
```

Comment:
Does this definitely work as `""`? It shouldn't be `/`?

Reply:
Looks like this was fixed in a later commit. Resolving.